### PR TITLE
LUN-40: Logic of single choice exercise

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/screens/SingleChoiceExerciseScreen.tsx [hunk 1]
```diff
@@ -25,42 +38,129 @@ const SingleChoiceExerciseScreen = ({ navigation, route }: SingleChoiceExerciseS
...
     axios.get(url).then(response => {
-      console.log('get', trainingSetId)
       setDocuments(response.data)
       setCount(response.data.length)
       setIsLoading(false)
     })
   }, [trainingSetId])
 
-  const buildAnswerOption = React.useCallback(() => {}, [currentWord, documents])
+  const generateFalseAnswers = React.useCallback(
```
Comment (klinzo) [inline src/screens/SingleChoiceExerciseScreen.tsx:54]: I'd move this to a helper method out of the component.

**Review by klinzo — APPROVED** [review]
There is an empty component: `ChoiceExercisesScreen`, is this intended?

File: src/screens/SingleChoiceExerciseScreen.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply src/screens/SingleChoiceExerciseScreen.tsx:53]: will do in the next ticket.

File: src/constants/data.ts [hunk 2]
```diff
@@ -70,6 +70,15 @@ export const ARTICLES = {
...
 export type Article = typeof ARTICLES[keyof typeof ARTICLES]
 
+export function isArticle(str: string): str is Article {
+  for (const article in ARTICLES) {
+    if (str === article) {
+      return true
+    }
+  }
+  return false
+}
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/constants/data.ts]: done

**Pull request merged**